### ISSUE TITLE
inetLatency fails on some non-English versions of Windows

### ISSUE THREAD

**Issue by GitSquared** [sebhildebrandt/systeminformation#204]
**Describe the bug**
The `ping` wrapper used for the Windows version of `inetLatency` relies on parsing specific English text that is translated in some non-English versions of Windows.

https://github.com/sebhildebrandt/systeminformation/blob/a9f346a69ae74cc40b947a73d739450e3b34b7a9/lib/internet.js#L163

The output it's expecting is as follows:
```
Approximate round trip times in milli-seconds:
    Minimum = 5ms, Maximum = 5ms, Average = 5ms
```

On a Spanish version of Windows for example, here's the actual output from `ping`:
```
Tiempos aproximados de ida y vuelta en milisegundos:
    Mínimo = 14ms, Máximo = 15ms, Media = 14ms
```

**Environment:**
 - systeminformation package version: Latest (v4.0.6)
 - OS: Windows 10
 - Hardware: Unrelated

**Additional context**
Issue was found downstream at GitSquared/edex-ui#414.

A possible fix could be to parse from `ms` instead of `min`, since the millisecond units are consistent across languages (I think).

**Comment by sebhildebrandt** [comment]
Ahh, ok, will fix this ... sorry!

**Comment by sebhildebrandt** [comment]
V4.0.7 just released ... can you try it?

**Comment by GitSquared** [comment]
@sebhildebrandt Seems to be working now. Thanks for the quick fix!

**Comment by RangerCD** [comment]
@sebhildebrandt V4.0.7 still doesn't cover Chinese situation, here is a typical Chinese output sample:
```
>ping 1.1.1.1 -n 1

正在 Ping 1.1.1.1 具有 32 字节的数据:
来自 1.1.1.1 的回复: 字节=32 时间=174ms TTL=53

1.1.1.1 的 Ping 统计信息:
    数据包: 已发送 = 1，已接收 = 1，丢失 = 0 (0% 丢失)，
往返行程的估计时间(以毫秒为单位):
    最短 = 174ms，最长 = 174ms，平均 = 174ms

```
Notice that a `，`(\uff0c Chinese comma) is not compatible with `,`(\u002c English comma) in https://github.com/sebhildebrandt/systeminformation/blob/ad0d1c6cc088e36622e10f73f351a184ea37fbce/lib/internet.js#L163
Maybe judge by `line.endsWith("ms")` would meet the needs?

**Comment by sebhildebrandt** [comment]
Very clear. Have already an idea for that ;-)

**Comment by sebhildebrandt** [comment]
So I published a fix (v4.0.8) that handles also Chinese output. Hope this works as expected.
